sdk/spring/azure-spring-boot/src/main/java/com/microsoft/azure/spring/autoconfigure/aad/AADOAuth2UserService.java

```diff
@@ -50,25 +45,44 @@ public OidcUser loadUser(OidcUserRequest userRequest) throws OAuth2Authenticatio
             // https://github.com/MicrosoftDocs/azure-docs/issues/8121#issuecomment-387090099
             // In AAD App Registration configure oauth2AllowImplicitFlow to true
             final ClientRegistration registration = userRequest.getClientRegistration();
-            final AzureADGraphClient graphClient = new AzureADGraphClient(
+            final AzureADGraphClient azureADGraphClient = new AzureADGraphClient(
                 registration.getClientId(),
                 registration.getClientSecret(),
-                aadAuthProps,
-                serviceEndpointsProps
+                aadAuthenticationProperties,
+                serviceEndpointsProperties
             );
-            String graphApiToken = graphClient
-                .acquireTokenForGraphApi(userRequest.getIdToken().getTokenValue(), aadAuthProps.getTenantId())
+            String graphApiToken = azureADGraphClient
+                .acquireTokenForGraphApi(
+                    userRequest.getIdToken().getTokenValue(),
+                    aadAuthenticationProperties.getTenantId()
+                )
                 .accessToken();
-            mappedAuthorities = graphClient.getGrantedAuthorities(graphApiToken);
+            mappedAuthorities = azureADGraphClient.getGrantedAuthorities(graphApiToken);
         } catch (MalformedURLException e) {
-            throw wrapException(INVALID_REQUEST, "Failed to acquire token for Graph API.", null, e);
+            throw toOAuth2AuthenticationException(
```

Comment:
this is just a style comment, but I personally find the original form of the throw (1 line versus 5 lines) easier to read.

Reply:
Hi, @joshfree .
Thank you for your feedback.
Here is my principal about whether to break line:
1. If line length <= 120, it should be written in 1 line.
2. If length > 120, write 1 parameter in one line, which will make code easier to read.

In this case, I renamed method name from `wrapException` to `toOAuth2AuthenticationException`, this will make this line's length > 120, so I write each parameter in a line.